mautic/mautic
# Focus: Remove timeout from success message after form send

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

If you send the form in a popup you see a success message in just 5 seconds and then you see the form again. This PR remove it and keep success message until user do not close window



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Create a popup with a form
3.  Submit the form from popup
4. See like you see an empty form after 5 seconds. It makes some users confused
5. After PR you should see a success message forever
